scripts/armstrong.py

```diff
@@ -0,0 +1,16 @@
+# armstrong number: 153 = 1^3+5^3+3^3
+
+def is_armstrong(n):
+    s=0
+    a = n
+    while(n>0):
+        p = n %10
+        s = s + (p*p*p)
+        n = int(n/10)
+    if s == a:
+        return True
+    else:
+        return False
+
+n = int(input("Enter any number: "))
+print(is_armstrong(n))
```

Comment:
Put this as follows:

```python

if __name__ == '__main__':
    n = int(input("Enter any number: "))
    print(is_armstrong(n))

```
Follow this [link](https://www.freecodecamp.org/news/if-name-main-python-example/#:~:text=We%20can%20use%20an%20if,name%20if%20it%20is%20imported.) to learn why we would want to do this. 
But basically two reasons:
1. if you run it as ```python armstrong.py``` -> We want to take user input
2. if you want to import is_armstrong function -> We DON'T want to take user input, rather just import ```is_armstrong``` function as: 
```python 
from armstrong import is_armstrong
```

Reply:
Implemented